src/pages/index.js

```diff
@@ -111,7 +111,7 @@ class Home extends Component {
                         fontSize: 60,
                       },
                     }}>
-                    React
+                    ری ایکٹ
```

Comment:
Do not translate the words that represent products i.e. React, JavaScript etc. Read the [universal style guide](https://github.com/reactjs/reactjs.org-translation/blob/master/style-guide.md) and see examples from other translation projects such as [React docs in Arabic](https://ar.reactjs.org).